README.md

```diff
@@ -78,7 +78,7 @@ You can also use `trace.out` in the root folder with `Go` default tracing UI
 ## Loki debug
 You can check all the messages the tool sends with env var `WASP_LOG_LEVEL=trace`
 
-If Loki client fail to deliver a batch test will proceed, if you experience Loki issues, consider setting `Timeout` in `LokiConfig` or set `IgnoreErrors: false` to fail the test on any error
+If Loki client fail to deliver a batch test will proceed, if you experience Loki issues, consider setting `Timeout` in `LokiConfig` or set `MaxErrors: 10` to fail the test after N errors
```

Comment:
Is this correct  `to fail the test after N errors`?  or should it rather say that `Loki` will try N times and then return error (or exit?)